governance/polls/atification Poll for Modify Core Unit Budget - Collateral Engineering Services, CES-001 (MIP40c3-SP57) - February 14, 2022.md

```diff
@@ -0,0 +1,62 @@
+summary: MIP40c3-SP57 is a new MIP providing for the continuation of the Dai budget implementation for Core Unit CES-001: Collateral Engineering Services.
+discussion_link: https://forum.makerdao.com/t/mip40c3-sp57-modify-core-unit-budget-collateral-engineering-services-ces-001/12578
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-02-14T16:00:00
+end_date: 2022-02-28T16:00:00
+---
+# Ratification Poll for Modify Core Unit Budget - Collateral Engineering Services, CES-001 (MIP40c3-SP57) - February 14, 2022
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, February 14 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP40c3-SP57 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*The MIP40c3-SP57 Dai budget implementation for Core Unit CES-001: Collateral Engineering Services contains:
+ The CES-001 budget for Q2 2022 through Q1 2023. This is a continuation of the Q4 2021 through Q1 2022 budget in MIP40c3-SP30 1.
+ Automates Dai payments from the Maker Protocol surplus buffer to the Auditor Wallet by establishing a DssVest stream for funding.
+ Unless specified in MIP40c3-SP57, all prior terms and conditions ratified in the prior Dai budget implementation, MIP40c3-SP30 1, are considered active and are not modified or revoked by this proposal.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/02fa322deac6cd38a48336f54ac475c2ef37b4b0/MIP40/MIP40c3-Subproposals/MIP40c3-SP57.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP40c3SP57)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip40c3-sp57-modify-core-unit-budget-collateral-engineering-services-ces-001/12578)
+
+## Outcomes
+
+This poll implements a **Minimum Positive Participation** value. The Minimum Positive Participation is currently set to **10,000 MKR**.
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option AND the votes for the 'Yes' option exceed 10,000 MKR, then the following actions will be taken:**
+* The MIP Editors will mark the proposal **Accepted** and the Governance Facilitators will confirm its passage on the Governance and Risk call on Thursday, February 24. 
```

Comment:
The date here is wrong, the vote doesn't finish until the 28th, so it would be confirmed on Thursday, March 3rd